Source/Core/Core/HW/WiimoteReal/IOLinux.cpp

```diff
@@ -137,28 +137,43 @@ bool WiimoteLinux::ConnectInternal()
   addr.l2_family = AF_BLUETOOTH;
   addr.l2_bdaddr = m_bdaddr;
   addr.l2_cid = 0;
+  int retry;
 
   // Output channel
   addr.l2_psm = htobs(WM_OUTPUT_CHANNEL);
-  if ((m_cmd_sock = socket(AF_BLUETOOTH, SOCK_SEQPACKET, BTPROTO_L2CAP)) == -1 ||
-      connect(m_cmd_sock, (sockaddr*)&addr, sizeof(addr)) < 0)
+  retry = 0;
```

Comment:
retry is set to zero right before the output channel and the input channel and having the declaration a few lines up looks fine, I think.